src/rollup-plugin-dependency-info.ts

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import path from 'path';
+import {OutputOptions, OutputBundle} from 'rollup';
+
+type Dependency = {
```

Comment:
Since `Dependency` / `Dependencies` / `DependencyType` are all data types that get exposed/exported to the rest of the program through `DependencyInfoOutput`, we need to give them less generic names so that we don't get confused.

`DependencyStats`, `DependencyStatsMap`, `DependencyStatsType`?